admin/src/index.js

```diff
@@ -7,14 +7,14 @@ const app = express()
 
 app.use(bodyParser.json({limit: "10mb"}))
 
-app.get("/people/:id", (req, res) => {
+app.get("/user/:id", (req, res) => {
   const {id} = req.params
-  request.get(`${config.serviceToCallUrl}/people/${id}`, (e, r, person) => {
+  request.get(`${config.holdings}/holdings/${id}`, (e, r, holdings) => {
```

Comment:
everything is named `holdings` here. useful for the config to be named `holdingsServiceUrl`

Reply:
yeah I agree with this